### ISSUE TITLE
Doesn't compile with D 2.067

### ISSUE THREAD

**Issue by Maaaks** [PhilippeSigaud/Pegged#155]
On Ubuntu 14.10, Pegged stop compiling after I upgraded D from 2.066.1 to 2.067.0. The reason seems to be this line in `peg.d`:
```d
import std.range: equal;
```
Could you please remove it? As far as I can see, it isn't used in the file anyway.

**Comment by PhilippeSigaud** [comment]
Did you use version 0.2.1? v0.1 used std.range: equal, but that was changed
Q4 2014, to prepare the way for 2.067. IIRC, dub now refers to v0.2.1

**Comment by Maaaks** [comment]
Oops, sorry, I really use the old version.

**Comment by PhilippeSigaud** [comment]
I changed it not too long ago, but forgot the update the version to have
dub catch it, so really it's my fault.

**Comment by Maaaks** [comment]
Mine too, since I hardcoded the version of Pegged in my dependencies because of backwardnoncompatibilityphobia :-) and forgot to check for updates manually. Anyway, thank you for the fact that it works and is updated properly!                                                                                                                                                                                                                                                                                                                                        --Макс Алибаевhttps://maaaks.ru/                                                                                                                                                                                                                From: Philippe SigaudSent: понедельник, 6 апреля 2015 г., 19:16To: PhilippeSigaud/PeggedReply To: PhilippeSigaud/PeggedCc: MaaaksSubject: Re: [Pegged] Doesn't compile with D 2.067 (#155)I changed it not too long ago, but forgot the update the version to have
dub catch it, so really it's my fault.

On Mon, Apr 6, 2015 at 7:53 AM, Maaaks <notifications@github.com> wrote:

> Oops, sorry, I really use the old version.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PhilippeSigaud/Pegged/issues/155#issuecomment-89934905>
> .
>


—Reply to this email directly or view it on GitHub.
